dali/base/dafdesc.cpp

```diff
@@ -2336,7 +2336,29 @@ IFileDescriptor *createFileDescriptor(const char *lname,IGroup *grp,IPropertyTre
     return res;
 }
 
+IFileDescriptor *createFileDescriptor(const char *lname, const char *clusterType, const char *groupName, IGroup *group)
+{
+    StringBuffer partMask;
+    unsigned parts = group->ordinality();
+    getPartMask(partMask, lname, parts);
+
+    StringBuffer curDir, defaultDir;
+    if (!getConfigurationDirectory(nullptr, "data", clusterType, groupName, defaultDir))
```

Comment:
could "data" be #defined and given a self documenting name so others will know the significance of "data"

Reply:
Will do.